### PR TITLE
🚀 Upgrade the world

### PR DESCRIPTION
**Pull request by JasonEtco**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -478,6 +452,6 @@ module.exports = {
         "transition",
         "animation",
       ],
-    } ],
+    }],
```
Comment (matchai) [inline index.js]: Starting closing bracket has a space before its brace.

Comment (JasonEtco) [inline index.js]: I don't like that tho

Comment (matchai) [inline index.js]: So remove the other spaces between brackets and braces. There are a few in the file.

Comment (JasonEtco) [inline index.js]: I fixed it for u <3

**Comment by JasonEtco** [comment]
Will do in a later PR (maybe probably not). For now I just want to :shipit:

**Pull request merged**